Makefile

```diff
@@ -1,6 +1,6 @@
 BIN_NAME=terraform-provider-instaclustr
 
-VERSION=2.0.48
+VERSION=2.0.51
 
```

Comment:
Seems we have missed bumping version to `49` . Looking at https://registry.terraform.io/providers/instaclustr/instaclustr/latest/docs it looks like `50` is the current version